extra/alacritty.man

```diff
@@ -57,6 +57,9 @@ Defines the window position. Falls back to position specified by window manager
 \fB\-t\fR, \fB\-\-title\fR <title>
 Defines the window title [default: Alacritty]
 .TP
+\fB\-\-embed\fR <parent>
+Sets the XEmbed parent window ID (as a long value)
```

Comment:
I don't think using `long` here is doing us any benefits, since it's more likely to confuse or help.

Just calling it a decimal integer is likely to be more user friendly. We could optionally also add a little more info to what this does, I think XTerm does a good job here: https://linux.die.net/man/1/xterm (search for 'embed')

Reply:
Ok.  Hopefully the current rendition is better.